convert2rhel/systeminfo.py

```diff
@@ -476,32 +508,30 @@ def _is_dbus_running(self):
         return status
 
     def get_system_release_info(self, system_release_content=None):
-        """Return the system release information as an dictionary
+        """Return the system release information as a dictionary
 
-        This function aims to retrieve the system release information in an dictionary format.
+        This function aims to retrieve the system release information in a dictionary format.
         This can be used before and after we modify the system-release file on the system,
-        as it have a parameter to to read from the contents of a system-release file (if called from somewhere else).
+        as it have a parameter to read from the contents of a system-release file (if called from somewhere else).
 
         :param system_release_content: The contents of the system_release file if needed.
         :type system_release_content: str
         :returns: A dictionary containing the system release information
         :rtype: dict[str, str]
         """
 
-        system_release_data = self.parse_system_release_content(system_release_content)
-
         release_info = {
-            "id": system_release_data["distribution_id"],
-            "name": system_release_data["name"],
-            "version": "%s.%s" % (system_release_data["version"].major, system_release_data["version"].minor),
+            "id": self.distribution_id,
+            "name": self.name,
+            "version": repr(self.version),
```

Comment:
As we now have repr for the class it should be possible to use the class itself, so it can be made into a string on-demand. But requires a bit more testing that might be out of scope, so up to you
```suggestion
            "version": self.version,
```

Reply:
Yeah I think out of scope